os/include/debug.h

```diff
@@ -817,6 +817,12 @@ int get_errno(void);
 #define dmallvdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_IRQ_INFO
+#define irqinfo(format, ...)   vdbg(format, ##__VA_ARGS__)
+#else
+#define irqinfo(...)
+#endif
+
```

Comment:
This needs to be defined at two points in the file. Please search for CONFIG_DEBUG_DMA_INFO and make similar change for IRQ_INFO in both locations.

Reply:
okay